modules/structs/repo.go

```diff
@@ -163,8 +163,6 @@ type EditRepoOption struct {
 type TransferRepoOption struct {
 	// required: true
 	NewOwner string `json:"new_owner"`
-	// ID of the team or teams to add to the repository. Teams can only be added to organization-owned repositories.
```

Comment:
why do you remove this function?

Reply:
if the actor is admin of the transfering repo AND the aim organisation he can asign Teams to it...

Reply:
Mapping the API behavior to the regular transfer via web

Reply:
@adelowo -> https://developer.github.com/v3/repos/#transfer-a-repository is having this too ...

Reply:
I think we can support repo-confirm & teamID ..
* If actor has right to create a repo and manage it on destination user (becauseo of admin rights or owner of an organisation) -> take kare of TeamID
* If actor has NO right to destination -> Ignore TeamID (perhaps a warning to log)